src/main/java/de/qabel/desktop/hockeyapp/CrashesClient.java

```diff
@@ -0,0 +1,7 @@
+package de.qabel.desktop.hockeyapp;
+
+import java.io.IOException;
+
+public interface CrashesClient {
```

Comment:
CrashReporterClient would be a better name, because it hopefully doesn't crash the client ;)